lib/hanami/model/error.rb

```diff
@@ -88,5 +88,14 @@ def initialize(message = 'Check constraint has been violated')
         super
       end
     end
+
+    # Unknown database type error for repository auto-mapping
+    #
+    # @since x.x.x
+    class UnknownDatabaseTypeError < Error
+      def initialize(message)
```

Comment:
UPDATE: I'm still shipping this PR, because we still want to increase the coverage of ROM exceptions. If `ROM.container` raises another error we'll be able to map it.